rero_ils/modules/documents/jsonschemas/documents/document_intended_audience-v0.0.1.json

```diff
@@ -0,0 +1,411 @@
+                    "value": "target_school_harmos4"
+                  },
+                  {
+                    "label": "target_school_harmos5",
+                    "value": "target_school_harmos5"
+                  },
+                  {
+                    "label": "target_school_harmos6",
+                    "value": "target_school_harmos6"
+                  },
+                  {
+                    "label": "target_school_harmos7",
+                    "value": "target_school_harmos7"
+                  },
+                  {
+                    "label": "target_school_harmos8",
+                    "value": "target_school_harmos8"
+                  },
+                  {
+                    "label": "target_school_harmos9",
+                    "value": "target_school_harmos9"
+                  },
+                  {
+                    "label": "target_school_harmos10",
+                    "value": "target_school_harmos10"
+                  },
+                  {
+                    "label": "target_school_harmos11",
+                    "value": "target_school_harmos11"
+                  },
+                  {
+                    "label": "target_school_upper_secondary",
+                    "value": "target_school_upper_secondary"
+                  },
+                  {
+                    "label": "target_school_tertiary",
+                    "value": "target_school_tertiary"
+                  },
+                  {
+                    "label": "target_school_bachelor",
+                    "value": "target_school_bachelor"
+                  },
+                  {
+                    "label": "target_school_master",
+                    "value": "target_school_master"
+                  }
+                ]
+              }
+            }
+          }
+        },
+        {
+          "title": "filmage_ch",
+          "propertiesOrder": [
+            "audienceType",
+            "value"
+          ],
+          "required": [
+            "audienceType",
+            "value"
+          ],
+          "type": "object",
+          "properties": {
+            "audienceType": {
+              "title": "filmage_ch",
+              "type": "string",
+              "readOnly": true,
+              "default": "filmage_ch",
+              "const": "filmage_ch",
+              "form": {
+                "wrappers": [
+                  "hide"
+                ]
+              }
+            },
+            "value": {
+              "title": "value",
+              "type": "string",
+              "default": "target_filmage_ch_12",
+              "enum": [
+                "target_filmage_ch_18",
+                "target_filmage_ch_16",
+                "target_filmage_ch_14",
+                "target_filmage_ch_12",
+                "target_filmage_ch_10",
+                "target_filmage_ch_7",
+                "target_filmage_ch_5",
+                "target_filmage_ch_2"
```

Comment:
Here we don't want the prefix. We want the following values:

from the age of 18
from the age of 16
from the age of 14
from the age of 12
from the age of 10
from the age of 7
from the age of 5
from the age of 2